php/php.editor/src/org/netbeans/modules/php/editor/elements/MethodElementImpl.java

```diff
@@ -259,8 +266,8 @@ public boolean isConstructor() {
     @Override
     public String getSignature() {
         StringBuilder sb = new StringBuilder();
-        sb.append(getName().toLowerCase()).append(Separator.SEMICOLON); //NOI18N
-        sb.append(getName()).append(Separator.SEMICOLON); //NOI18N
+        sb.append(getName().toLowerCase()).append(Separator.SEMICOLON); // 0: lower case name
```

Comment:
Nitpick: `toLowerCase()` should not be called without any argument, right?

Reply:
Should probably be `toLowerCase(Locale.ROOT)` in this case. Good catch (and not a nitpick IMO), although I think we have a lot of these!

Reply:
Totally agree :+1:

Reply:
You are right! Will fix it. Thanks!

Reply:
Ffixed.